translation-tests/src/integration-test/java/eu/europeana/api/translation/tests/web/TranslationRestIT.java

```diff
@@ -113,13 +115,22 @@ void translateErrorNoTarget() throws Exception {
         + "\"source\": \"de\","
         + "\"text\": [ \"eine Textzeile auf Deutsch\"]"
         + "}";
-    mockMvc
+    String response = mockMvc
         .perform(
             post(BASE_URL_TRANSLATE)
               .header(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON_VALUE)
               .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
               .content(missingTarget))
-        .andExpect(status().isBadRequest());
+        .andExpect(status().isBadRequest())
+        .andReturn().getResponse().getContentAsString();
+    
+    JSONObject obj = new JSONObject(response);
```

Comment:
The jackson ObjectMapper should be used here to parse the response to EuropeanaApiErrorResponse object, using a parseErrorResponse() method